src/extractor/extraction_containers.cpp

```diff
@@ -53,6 +112,12 @@ ExtractionContainers::ExtractionContainers()
     name_lengths.push_back(0);
     name_lengths.push_back(0);
 
+    // Insert three zero offsets for empty strings and the total length
```

Comment:
this comment needs to be adjusted, if we push four zeros. It also requires an explanation of what the different indices are supposed to be.